src/hotspot/share/gc/shenandoah/shenandoahCollectionSet.hpp

```diff
@@ -36,7 +36,7 @@ class ShenandoahCollectionSet : public CHeapObj<mtGC> {
   friend class ShenandoahHeap;
   friend class ShenandoahCollectionSetPreselector;
 
-  void establish_preselected(bool *preselected) {
+  void establish_preselected(bool preselected[]) {
```

Comment:
Are we going with `bool* preselected` now?

Reply:
Thanks for finding that.  Fixed.